README.md

```diff
@@ -46,15 +46,22 @@ If you would not, these installation are not required.
 
 ### Bootstraping container filesystem
 
+Create mount points:
+
+```sh
+$ mkdir -p /var/another/root/etc
+$ mkdir -p /var/another/root/home
+```
+
 Create the file `example.haco`:
 
 ```ruby
-Haconiwa::Base.define do |config|
+Haconiwa::Base do |config|
```

Comment:
So, now i use `Haconiwa.define`
https://github.com/haconiwa/haconiwa/blob/af3d22c822baea165bd825cda0f04f39c25bbc7e/mrblib/haconiwa/generator.rb#L5

Reply:
Oh, I mistook the correction...